indexer/src/test/java/org.pdxfinder/commands/TestConfig.java

```diff
@@ -0,0 +1,26 @@
+package org.pdxfinder.commands;
```

Comment:
Same here... strange file path includes a dot.  Java packages are usually namespaced corresponding to directory structure